src/responses.rs

```diff
@@ -312,3 +312,66 @@ impl TryFrom<Arc<matrix_sdk_common::deserialized_responses::EncryptionInfo>> for
         }
     }
 }
+
+/// Struct containing information on how a to-device message was decrypted.
+#[wasm_bindgen]
+#[derive(Debug, Clone)]
+pub struct ToDeviceEncryptionInfo {
+    /// The base64-encoded public Curve25519 key of the device that encrypted
+    /// the message.
+    #[wasm_bindgen(getter_with_clone, js_name = "senderCurve25519Key")]
+    pub sender_curve25519_key_base64: String,
+
+    /// The user ID of the sender of the event.
+    ///
+    /// Note this is untrusted data unless {@link #isSenderVerified} is true.
+    #[wasm_bindgen(getter_with_clone)]
+    pub sender: identifiers::UserId,
+
+    /// The device ID of the device that sent us the to-device message.
+    ///
+    /// Could be `undefined` in the case where the to-device message sender
+    /// checks are delayed. There is no delay for to-device messages other
+    /// than `m.room_key`, so this will not always be truthy for other
+    /// message types (the decryption would fail if the sender device keys
+    /// cannot be found).
```

Comment:
I'm having trouble understanding this. Should the "not" be there?

Reply:
no, good catch, thanks.